packages/application-shell/src/apollo-links/header-link.ts

```diff
@@ -62,6 +66,30 @@ const getAppliedForwardToHeaders = (
   return {};
 };
 
+const extractSessionTokenFromResponse = (
+  context: ApolloContextAfterResponse
+): string | null => {
+  const refreshedSessionToken = context.response?.headers?.get(
+    'x-refreshed-session-token'
+  );
+  if (refreshedSessionToken) {
+    return refreshedSessionToken ?? null;
```

Comment:
It is not clear to me why we are using the nullish coalescing operator since the if statement above already checked if `refreshedSessionToken` exist.

Reply:
Yes simply because the value could be `undefined`, but the function explicitly expects to return `null`.